ex_app/lib/main.py

```diff
@@ -140,8 +144,10 @@ async def init_callback(b_tasks: BackgroundTasks, nc: typing.Annotated[Nextcloud
     return responses.JSONResponse(content={})
 
 
+APP_ENABLED = False
```

Comment:
this will be out of sync when container restarted.

We have added an new endpoint to AppAPI for this: https://github.com/cloud-py-api/app_api/pull/318